notes/zrq/20210729-02-redeploy-2100622.txt

```diff
@@ -0,0 +1,1103 @@
+    >     "title": "Raw catalogue with selected columns",
+    >     "result": "SUCCESS",
+    >     "time": "0:5:59"
+    >   }
+    >   {
+    >     "title": "Visualisation (colour / absolute-magnitue diagram) of the raw catalogue",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:5"
+    >   }
+    >   {
+    >     "title": null,
+    >     "result": "SUCCESS",
+    >     "time": "0:0:0"
+    >   }
+    >   {
+    >     "title": "Define the training samples",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:5"
+    >   }
+    >   {
+    >     "title": "Assemble training and reserve test sets",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:0"
+    >   }
+    >   {
+    >     "title": "Train up the Random Forrest",
+    >     "result": "SUCCESS",
+    >     "time": "0:4:3"
+    >   }
+    >   {
+    >     "title": "Check feature set for nulls",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:0"
+    >   }
+    >   {
+    >     "title": "Classify the reserved test sets",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:0"
+    >   }
+    >   {
+    >     "title": "Classification confusion matrix",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:10"
+    >   }
+    >   {
+    >     "title": "Relative importance of the selected features",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:0"
+    >   }
+    >   {
+    >     "title": "Apply the classification model and plot sample results",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:21"
+    >   }
+    >   {
+    >     "title": "Histogram of classification probability",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:30"
+    >   }
+    >   {
+    >     "title": "Sky distribution of good source sample",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:6"
+    >   }
+    >   {
+    >     "title": "Sky distribution of bad source sample",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:6"
+    >   }
+    >   {
+    >     "title": "Further reading and resources",
+    >     "result": "SUCCESS",
+    >     "time": "0:0:0"
+    >   }
+
+    >   0:11:28
+
+
+    #
+    # This is where we got to with the live deploy ...
+    # (actually, the live deploy ran the MK RandomForest twice)
+    #
+
+    #
+    # Login to the UI as 'zrq' and try running the source counts notebook.
+    # PASS - lovley blue/green/yellow sky plot :-)
+    #
+
+    #
+    # Try running the mean proper motions notebook.
+    # PASS - lovley ranbow plot of motions ;-)
+    #
+
+    #
+    # Try running the ML RandomForest notebook.
+    # It is running, but slowly.
+    # This is the first time I've run this notebook, but it has already been run once by our test user.
+    # Interpreter was reset by 'zrq', then the notebook was run by test users, now notebook is slow for 'zrq'.
+    # 1:7:23 to run
+    #
+
+    #
+    # These tests were all done on the 20210620-zrq-resources deployment.
+    # Shared interpreters, shared Spark context.
+    #
+
```

Comment:
Is this the dev system that Dennis is using currently? Was the interpreter cleared before he used it?